arbitrum-docs/node-running/how-tos/running-an-orbit-node.mdx

```diff
@@ -20,57 +20,102 @@ This how-to provides step-by-step instructions for running an Orbit node on your
+
+:::
+
+#### L3 Orbit chain info and chain name
+
+Providing the following flag is mandatory:
+  
+```bash
+  --chain.info-json=<Orbit Chain's chain info>
+```
+    
+The chain info json string contains vital information about the Orbit chain, which is necessary for the node to commence operations. An example of the chain info json string is available in the subsequent sections.
+    
+You need to provide:
+
+```bash
+--chain.name=<My Arbitrum L3 Chain>
+```
+
+And it must match the chain name that is used in `--chain.info-json` content.
+
+
+For an Anytrust chain, you will need to provide 
+
+```bash
+--node.data-availability.enable
+  ```
+
+#### Important ports
 
 - RPC: `8547`
 - Sequencer Feed: `9642`
 - WebSocket: `8548`
   - WS port `8548` needs extra args to be opened. Please use these flags:
-    - --ws.port=8548
-    - --ws.addr=0.0.0.0
-    - --ws.origins=\*
+    - `--ws.port=8548`
+    - `--ws.addr=0.0.0.0`
+    - `--ws.origins=\*`
 
 ### Putting it all together
 
-- When running docker image, an external volume should be mounted to persist the database across restarts. The mount point inside the docker image should be `/home/user/.arbitrum`
-- Here is an example of how to run a node for an Orbit chain:
+When running docker image, an external volume should be mounted to persist the database across restarts. The mount point inside the docker image should be `/home/user/.arbitrum`
 
-  - Note that is important that `/some/local/dir/arbitrum` already exists, otherwise the directory might be created with `root` as owner, and the docker container won't be able to write to it
+Here is an example of how to run a node for an Orbit chain, Note that is important that `/some/local/dir/arbitrum` already exists, otherwise the directory might be created with `root` as owner, and the docker container won't be able to write to it:
 
-  ```shell
-  docker run --rm -it  -v /some/local/dir/arbitrum:/home/user/.arbitrum -p 0.0.0.0:8547:8547 -p 0.0.0.0:8548:8548 @latestNitroNodeImage@ --parent-chain.connection.url=<Parent chain RPC URL> --chain.id=<L3ChainId> --chain.name=<My Arbitrum L3 Chain> --http.api=net,web3,eth --http.corsdomain=* --http.addr=0.0.0.0 --http.vhosts=* --chain.info-json=<Orbit Chain's chain info>
+```bash
+docker run --rm -it  -v /some/local/dir/arbitrum:/home/user/.arbitrum -p 0.0.0.0:8547:8547 -p 0.0.0.0:8548:8548 @latestNitroNodeImage@ --parent-chain.connection.url=<Parent chain RPC URL> --chain.id=<L3ChainId> --chain.name=<My Arbitrum L3 Chain> --http.api=net,web3,eth --http.corsdomain=* --http.addr=0.0.0.0 --http.vhosts=* --chain.info-json=<Orbit Chain's chain info>
+```
 
-  ```
+When using the flag `--chain.info-json=<Orbit Chain's chain info>`, it is essential to replace `Orbit Chain's chain info` with the specific `chain info` json string of the Orbit chain for which you wish to run the node. For your convenience, an example of an chain info json string is provided below:
```

Comment:
```suggestion
When using the flag `--chain.info-json=<Orbit Chain's chain info>`, it is essential to replace `Orbit Chain's chain info` with the specific `chain info` json string of the Orbit chain for which you wish to run the node. For your convenience, an example of a chain info json string is provided below:
```

Reply:
Thanks @mahsamoosavi!
 I mistakenly committed this article. I will create a specific PR for it and include your suggestions.